solutions/swb-ui/src/api/environments.ts

```diff
@@ -1,19 +1,19 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 /*
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
 
+import { convertToRecord, httpApiGet, httpApiPut, httpApiPost } from '@aws/workbench-core-swb-common-ui';
 import useSWR from 'swr';
 import {
   EnvironmentItem,
   EnvironmentConnectResponse,
   CreateEnvironmentForm,
   EnvironmentsQueryParams
 } from '../models/Environment';
-import { httpApiGet, httpApiPut, httpApiPost } from './apiHelper';
-import { convertToRecord } from '../common/utils';
 
-const useEnvironments = (params?: EnvironmentsQueryParams) => {
+const useEnvironments = (params?: EnvironmentsQueryParams): any => {
```

Comment:
Is using `any` required here? It looks like the function has a defined return type.

Reply:
The type `any` here is correct (the environment output field will need an eslint ignore statement anyways for `any` type)